SwiftLeeds/Views/My Conference/TalkCell.swift

```diff
@@ -32,14 +34,38 @@ struct TalkCell: View {
 
             VStack(alignment: .leading, spacing: 8) {
                 if let speaker = speaker {
-                    VStack(alignment: .leading, spacing: 4) {
-                        Text(speaker)
-                            .font(.headline.weight(.medium))
+                    HStack {
+                        VStack(alignment: .leading, spacing: 4) {
+                            Text(speaker)
+                                .font(.headline.weight(.medium))
 
-                        if let company = company {
-                            Text(company)
-                                .font(.subheadline.weight(.medium))
-                                .opacity(0.6)
+                            if let company = company {
+                                Text(company)
+                                    .font(.subheadline.weight(.medium))
+                                    .opacity(0.6)
+                            }
+                        }
+
+                        Spacer()
+
+                        if let imageURL = imageURL {
+                            AsyncImage(
+                                url: URL(string: "\(BaseURL.image)\(imageURL)"),
```

Comment:
It feels really weird to have the view do this mapping.

Ideally we'd take the API responses and map them into local entities, but given we're doing it this way, we could simply pretend the baseURL at decode time?

Reply:
I think we should have the whole URL from the API, I had to look at the web code just to get that vase URL into the App.  What happens if we decide to use a different endpoint in the future, it would stop working.  So this was more a temporary option for now.

Reply:
Good call! we can make that change on the backend pretty easily